src/main/java/ui/View.java

```diff
@@ -22,7 +23,7 @@ public void prepareGUI() {
         JFrame frame = new JFrame(TITLE);
         frame.setLayout(new BorderLayout());
         frame.setSize(WIDTH, HEIGHT);
-        frame.setResizable(false);
+        frame.setResizable(false); //set to false before
```

Comment:
not sure comment is needed here

Reply:
this was from before, i was testing the UI so i changed it to true